aries-site/src/pages/components/checkbox.mdx

```diff
@@ -36,8 +37,22 @@ Checked and unchecked states refer to when an individual CheckBox is checked or
 
 ### CheckBox outside of FormField
 
+When using CheckBox outside of Formfield, it does not have a border or label. The hover indentifier will also be on the CheckBox itself.
+Here are some common use cases for when a CheckBox can be used outside of a FormField:
+1. When there is progressive disclosure: i.e. clicking on the checkbox will reveal related options that wouldn't have been displayed unless the user checked the box.
```

Comment:
i.e. means "in other words"
e.g. means "for example"

I believe we want "for example" here because we are providing an example of progressive disclosure.

Personally, I think "progressive disclosure" is still too narrow and would prefer [my suggestion](https://github.com/grommet/hpe-design-system/pull/2971#discussion_r997479711) speaking to the checkbox is serving as a UI control. Medium-soft opinion.

```suggestion
1. There is progressive disclosure (e.g. clicking on the checkbox will reveal related options that wouldn't have been displayed unless the user checked the box).
```

Reply:
that's fair Kenny and I did also chat about being more detailed

Reply:
done